src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -196,4 +193,23 @@ public ResponseEntity<Void> deleteLineTypesCatalog() {
         lineTypesCatalogService.deleteLineTypesCatalog();
         return ResponseEntity.ok().build();
     }
+
+    @PostMapping(value = "/network-modifications/voltage-init", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Voltage init should not be part of this API, the modif is more generic and is just a particular use of generator table motif

Reply:
renamed